File: CPP_programming/Day6_Reusing classes/Person/Person.cpp
  #include"Person.h"

Person::Person()////////////////////////constructors
{

}
Person::Person(Cstring nm, Date bd):name(nm),b_date(bd)
{
    //Cstring(nm);
    //Date(bd);
}
Person :: Person(const char * nm, int d, int m, int y)//:name(nm),b_date(d,m,y)
{
    // Cstring(nm); //some error
    // Date(d,m,y);
}
Person::Person(Person &obj):name(obj.name), b_date(obj.b_date)
{
    // name = obj.name;
    // b_date = obj.b_date;
}
void Person::display()/////////////////////////facilitators
{
    name.display();
    b_date.display();
}

File: Core_java_new/_09Interface/A3_Employee_usingInterface/employee/Person.java
package employee;


public class Person {

	private String name;
	private Date bdate;
	public Person() {
		//super();
		name="null";
		bdate=new Date();
	}
	public Person(String name, int dd,int mm,int yy) {
		super();
		this.name = name;
		this.bdate = new Date(dd,mm,yy);
	}
	public Person(Person p) {
		super();
		this.name = p.name;
		this.bdate = p.bdate;
	}
	
	public void display()
	{
		System.out.println("Name: "+name);
//		System.out.println("Date: "+bdate);
		bdate.display();
	}
}